README.md

```diff
@@ -35,6 +36,38 @@ required for testing citus.
 * [Advanced fab Usage](#advanced-fab)
   * [Using Multiple Citus Installations](#multiple-installs)
 
+## <a name="running-automated-tests"></a>Running Automated Tests
+
+**Depending of the tests you trigger here, you can block at most 3 jobs slots in circleci for around 3 hours. Choose wisely the time you want to run the tests to not block development**
+
+If you want, you can run trigger a job which can run pgbench, scale and tpch tests. What the job does is:
+
+* It creates a cluster with the test resource group name
+* It connects to the coordinator
+* It runs the corresponding test for the job
+* It deletes the cluster.
+
+There is a separate job for each test and you can run any combinations of them. To trigger a job, you should create a branch which has specific prefixes.
+
+If the branch has a prefix pgbench, then pgbench job will be triggered.
```

Comment:
Suggestion: Instead of `all_test` maybe `all_performance_tests` ?

Reply:
Sure we can do that, I will need to do that right before merging so that I dont need to create a separate PR now ( as we have the conversations in this one)
I am adding as to do for right before merging to the description.